src/util/nestedArray/nestedArray.ts

```diff
@@ -0,0 +1,22 @@
+export function nestedIndexOf(hay: any[][], needle: any[]): number {
+	let i = 0;
+
+	hay.forEach((e: any) => {
+		if (e === needle { return i; }
+		i = i + 1;
+	});
+
+	return -1;
+}
```

Comment:
not quite sure what this method is doing differently to `Array.includes`.

Reply:
It is supposed to handle and return the index of an array in a nested array since javascript natively doesn't handle nested arrays as expected. I will try the other solutions you have mentioned in the call with the lodash library and the filter methode.